PoissonRegression.ipynb

```diff
@@ -0,0 +1,5371 @@
+{
```

Comment:
<p>Add a semicolon at the end of the `sample` call (`chain = mapreduce(. . .);`) so the chain description doesn't print at the end of the output log. You should add some text in a new cell below this saying </p><p><br></p><p>"We can take a look at our parameter estimates using `describe(chain)`:</p><p><br></p><p>And then include a code cell where you call `describe(chain)`. It's important to separate out sampling and looking at the chain so the whole document is easier to read.</p>

 _Reply via <a href='https://app.reviewnb.com/TuringLang/TuringTutorials/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Added semicolon, the text and the code for describe(chains) as you mentioned. Looks neater than before.</p>